.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor/
+/cache/
 /config/local/
 .idea
 composer.lock
```

Comment:
dlaczego `composer.lock` jest w gitignorze? To jest proszenie się o kłopoty.

QA testuje paczke w wersji 5.7.6, wszystko jest gotowe i działa. Robimy deploy w między czasie ktoś przez przypadek wydał zjebane 5.7.7 w związku z czym na produkcji zainstalowało się 5.7.7 -> efekt 500.

composer.lock w gitignorze powinien być tylko i wyłącznie w projektach typu SDK które się powinny dostosować do paczek jakie są zainstalowane w projekcie matce.